lib/ddtrace/contrib/grape/configuration/settings.rb

```diff
@@ -24,6 +25,13 @@ class Settings < Contrib::Configuration::Settings
           end
 
           option :service_name, default: Ext::SERVICE_NAME
+
+          option :error_statuses do |o|
+            o.default { Datadog::Ext::HTTP::ERROR_RANGE.to_a }
+            o.setter do |new_value, _old_value|
+              Datadog::Contrib::StatusCodeMatcher.new(new_value)
+            end
```

Comment:
Unfort we need to default to nil here to ensure we dont introduce a breaking change

```suggestion
          option :error_statuses do |o|
            o.default { nil }
            o.setter do |new_value, _old_value|
              return if new_value.nil?
              Datadog::Contrib::StatusCodeMatcher.new(new_value)
            end
```